src/app/functions/sendPageviewEvent.js

```diff
@@ -13,7 +13,7 @@ const sendPageviewEvent = (location, title = false) => {
         data: {}
     };
 
-    payload.data.page           = window.bluehostWpAdminUrl + '?page=bluehost#' + location.pathname; // full url
+    payload.data.page           = window.bluehostWpAdminUrl + 'admin.php?page=bluehost#' + location.pathname; // full url
```

Comment:
@0aveRyan Any reason we don't just use `location.href` here? It includes the hash.

Reply:
Ah, we're not including the domain.

Reply:
@wpscholar should be full URL... `admin_url()` will include domain. We chop it up on the Hiive side when we send to GA, but we send the event from WP with the fully-qualified URL.

Yes, I build the URL from scratch so that other query parameters don't get sent and improperly segment pageviews, i.e...
```
domain.com/wp-admin?page=bluehost&epc-purge-yadayada&some-other-param=1#/home/onboarding
```